internal/pkg/model/relation.go

```diff
@@ -45,7 +45,8 @@ type Relation interface {
 	Key() string                              // unique key for sorting and comparing
 	ParentKey(relationOwner Key) (Key, error) // if relation type is parent <-> child, then parent key is returned, otherwise nil
 	OtherSideKey(owner Key) Key               // get key of the other side
-	IsOwningSide() bool                       // if true, relation will be stored in the manifest
+	IsDefinedInManifest() bool                // if true, relation will be present in the manifest
+	IsDefinedInApi() bool                     // if true, relation will be present in API calls
```

Comment:
`IsOwningSide` -> premenovane na `IsDefinedInManifest`
Pridana metoda `IsDefinedInApi`.

**Vysvetlim:**
- Mame relaciu `VariablesValuesFROM Relation`, ktora sposobi, ze do konfiguracie sa pri ukladani do API prida `variables_id` (`IsDefinedInManifest = false`, `IsDefinedInApi = true`)
- A mame relaciu `VariablesValuesFOR Relation`, ktora sposobi, ze `variables` konfiguracia je ulozena v sub-dir `variables`, pod konfiguraciu, ktora `variables` pouziva, tato informacia je ulozena v `manifest.json` (`IsDefinedInManifest = true`, `IsDefinedInApi = false`)
- Jedna sa vlastne o 2 strany tej iste relacia, .... lokalne sa to uklada ku `variables` konfiguracii, ale v API je to ulozene pri konfiguracii co `variables` pouziva

**Pri `schedules` je situacia ina:**
- Vznikne relacia `SchedulerFor`, .... no na strane API aj lokalne bude ulozena na strane `keboola.scheduler` konfiguracie, tj. je to pod-adresar + v API obsahuje `configurationId` konfiguracie, ktoru ma spustit (`IsDefinedInManifest = true`, `IsDefinedInApi = true`)

Aby to bolo mozne reprezentovat, tak som pridal metodu `IsDefinedInApi`.
O tom je cely PR.